easybuild/easyconfigs/p/pIRS/pIRS-2.0.2-gompi-2019b.eb

```diff
@@ -0,0 +1,68 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'ConfigureMake'
+
+name = 'pIRS'
+version = '2.0.2'
+
+homepage = 'https://github.com/galaxy001/pirs'
+description = """pIRS (profile based Illumina pair-end Reads Simulator) is a program for simulating paired-end reads
+from a reference genome. It is optimized for simulating reads similar to those generated from the Illumina platform."""
+
+toolchain = {'name': 'gompi', 'version': '2019b'}
+toolchainopts = {'cstd': 'c++11'}
+
+# https://github.com/galaxy001/pirs
+github_account = 'galaxy001'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s_fix-Makefiles.patch']
+checksums = [
+    'f80051819ee2c4307ad7ff74d35785c6ba864e8219673e6b0f12ff326eedb2e4',  # v2.0.2.tar.gz
+    '5168b02abc5162b8c4175d76ee50f1bf876911b31449d21008c3c3464935d47f',  # pIRS-2.0.2_fix-Makefiles.patch
+]
+
+dependencies = [
+    ('Perl', '5.30.0'),
+    ('zlib', '1.2.11'),
+    ('Boost', '1.71.0'),
+    ('gzip', '1.10'),
+    ('gnuplot', '5.2.8'),
+    ('SAMtools', '0.1.20'),
+    ('Coreutils', '8.32'),
+]
+
+skipsteps = ['configure', 'install']
+
+unpack_options = '--strip-components=1'
+buildininstalldir = True
+
+buildopts = 'CC="$CC" CXX="$CXX" CFLAGS="$CFLAGS" CXXFLAGS="$CXXFLAGS" PKGDATADIR="%(installdir)s"'
+
+modextrapaths = {'PATH': ''}
+
+local_bins = ['gc_coverage_bias', 'pirs']
+local_perl_files = [
+    'alignment_stator', 'baseCalling_Matrix_analyzer', 'baseCalling_Matrix_calculator',
+    'baseCalling_Matrix_merger', 'gc_coverage_bias_plot', 'indelstat_sam_bam'
+]
+sanity_check_paths = {
+    'files': ['%s' % f for f in local_bins + local_perl_files],
+    'dirs': []
+}
+
+sanity_check_commands = [
+    # Binaries
```

Comment:
Maybe add a comment to explain why `grep` is used?

Probably because using `-h`/`--help` results in non-zero exit code?